packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -15,10 +15,18 @@ const propTypes = {
    */
   inputAttributes: PropTypes.object,
   /**
+   * Indicates whether or not the picker should be displayed after rendering.
+   */
+  isForceShowPicker: PropTypes.bool,
```

Comment:
The wording of this seems a little odd. Maybe isPickerForceShown?

Reply:
the prefix of 'is' is throwing me off for this name, I think just calling it forceShowPicker would work better

Reply:
I renamed to shouldShowPicker. https://github.com/cerner/terra-core/pull/974/commits/f120301502e3d390e72a669fce130604de572ae5